package.json

```diff
@@ -29,22 +30,15 @@
     "chromatic": "npx chromatic --project-token dqq8k95raoo"
   },
   "peerDependencies": {
-    "emotion-theming": "^10.0.27",
```

Comment:
this is a 100% dependency, we shouldn't rely on peers on the styling system

Reply:
Since we are using emotion-theming in hopsworks-front it's wise to have it in peerDependencies here, so that it will be installed only once. Just like we added React and React-dom to peers

Reply:
yeah, but then locally you have to use finicky solutions like `npm-install-peers` to be able to run your storybooks

and like if we have `quartz` as a standalone UI package, I find it quite reckless to depend on an external library to make use of its core functionality

we depend on `emotion-theming` only for `useTheme` in `hopsworks-front`
what I suggest is then we export `useTheme` from here and use it like
`import { useTheme } from '@logicalclocks/quartz` then
it would be much more straightforward and we'd not need the `emotion-theming` dependency anymore

wdyt?

Reply:
Right but it's the recommended solution to keep the bundle size minimal, otherwise we can include React and React dom as dependencies since we heavily rely on it to render quartz.

Yes if we just depend on useTheme that's a feasible solution to export the useTheme.

Reply:
so then I'm going with exporting `useTheme`

Reply:
and concerning bundle size: `emotion-theming` is `0.25 kb`
I don't think this is a game-changer

also, the whole thing I've done here reduced bundle size by like 30% anyways
so as a result, we're still much much smaller after all of these

and yeah, I agree that bundle size is something that really matters
we'll keep working on this in future